nx.json

```
@@ -125,6 +125,7 @@
         "tree-table"
       ]
     },
+    "elements": {},
```

you have to specify tags:

```suggestion
    "elements": {
      "tags": ["scope:elements", "type:library"]
    },
```